history-of-the-quakers-v4/original/13-book-7-chapter-7.adoc

```diff
@@ -205,6 +222,9 @@ A petition was accordingly drawn up in the name of the prisoner,
 setting forth the matter of fact, and consequent proceedings thereupon,
 and further alleged,
 
+[.embedded-content-document.letter]
+--
+
 that she had no intention of disturbing the public worship;
```

Comment:
Should I use ellipses here again?

Reply:
yeah, i think so. `&hellip;that she had <etc>`